apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -39,10 +40,22 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 	@Property()
 	draggable = true;
 
+	@Property({ nullable: true })
+	completionDate?: Date;
```

Comment:
we define the prop here, but not in the interface ITaskCard above
basically, we want to have it in the interface first, and given the interface is implemented in the entity, it will be required